packages/core/src/Stores/notification-store.js

```diff
@@ -430,14 +433,41 @@ export default class NotificationStore extends BaseStore {
         }
     }
 
+    @action.bound
     showCompletedOrderNotification(advertiser_name, order_id) {
         const notification_key = `order-${order_id}`;
 
+        const notification_redirect_action =
+            routes.cashier_p2p === window.location.pathname
+                ? {
+                      onClick: () => {
+                          this.p2p_order_props.redirectToOrderDetails(order_id);
+                          this.setP2POrderProps({
+                              ...this.p2p_order_props,
+                              order_id,
+                          });
+                          if (this.is_notifications_visible) this.toggleNotificationsModal();
+                          this.refreshNotifications();
+                      },
+                      text: localize('Give feedback'),
+                  }
+                : {
+                      route: `${routes.cashier_p2p}?order=${order_id}`,
+                      text: localize('Give feedback'),
+                  };
```

Comment:
When the user is within p2p, if they click give feedback, it doesn't navigate them to the order details page of that order. Hence I needed to take the redirectToOrderDetails function from p2p to redirect them to that page in p2p.